_inc/lib/admin-pages/class-jetpack-about-page.php

```diff
@@ -0,0 +1,676 @@
+							<?php $this->display_gravatars(); ?>
+						</ul>
+						<p class="meet-the-team">
+							<a href="https://automattic.com/about/" target="_blank" rel="noopener noreferrer" class="jptracks" data-jptracks-name="jetpack_about_meet_the_team"><?php esc_html_e( 'Meet the Automattic team', 'jetpack' ); ?></a>
+						</p>
+					</div>
+
+					<div class="jetpack-about__text">
+						<p>
+							<?php esc_html_e( 'We are the people behind WordPress.com, WooCommerce, Jetpack, Simplenote, Longreads, VaultPress, Akismet, Gravatar, Crowdsignal, Cloudup, and more. We believe in making the web a better place.', 'jetpack' ); ?>
+							<a href="https://automattic.com/" target="_blank" rel="noopener noreferrer" class="jptracks" data-jptracks-name="jetpack_about_learn_more">
+								<?php esc_html_e( 'Learn more about us.', 'jetpack' ); ?>
+							</a>
+						</p>
+						<p>
+							<?php esc_html_e( 'We’re a distributed company with over 875 Automatticians in more than 67 countries speaking at least 83 different languages. Our common goal is to democratize publishing so that anyone with a story can tell it, regardless of income, gender, politics, language, or where they live in the world.', 'jetpack' ); ?>
+						</p>
+						<p>
+							<?php esc_html_e( 'We believe in Open Source and the vast majority of our work is available under the GPL.', 'jetpack' ); ?>
+						</p>
+						<p>
+							<?php
+								// Maybe use printf() because we'll want to escape the string but still allow for the link, so we can't use esc_html_e().
+								echo wp_kses(
+									__( 'We strive to live by the <a href="https://automattic.com/creed/" target="_blank" class="jptracks" data-jptracks-name="jetpack_about_creed" rel="noopener noreferrer">Automattic Creed</a>.', 'jetpack' ),
+									array(
+										'a' => array(
+											'href'   => array(),
+											'class'  => array(),
+											'target' => array(),
+											'rel'    => array(),
+											'data-jptracks-name' => array(),
+										),
+									)
+								);
+							?>
+						</p>
+						<p>
+							<a href="https://automattic.com/work-with-us" target="_blank" rel="noopener noreferrer" class="jptracks" data-jptracks-name="jetpack_about_work_with_us">
+								<?php esc_html_e( 'Come work with us', 'jetpack' ); ?>
+							</a>
+						</p>
+					</div>
+				</div>
+			</div>
+
+			<div class="jetpack-about__colophon">
+				<h3><?php esc_html_e( 'Popular WordPress services by Automattic', 'jetpack' ); ?></h3>
+				<ul class="jetpack-about__services">
+				<?php $this->display_plugins(); ?>
+				</ul>
+
+				<p class="jetpack-about__services-more">
+				<?php
+				echo wp_kses(
+					__( 'For even more of our WordPress plugins, please <a href="https://profiles.wordpress.org/automattic/#content-plugins" target="_blank" rel="noopener noreferrer" class="jptracks" data-jptracks-name="jetpack_about_wporg_profile">take a look at our WordPress.org profile</a>.', 'jetpack' ),
```

Comment:
We should clean up the plugins in this profile. There's a lot of older ones that we don't maintain anymore that should probably be closed. Non-blocking.